dts/bindings/mtd/jedec,spi-nor.yaml

```diff
@@ -24,3 +24,28 @@ properties:
     type: phandle-array
     required: false
     description: RESETn pin
+  sfdp-basic-8:
+    type: int
+    required: false
+    default: 0xD810200C
+    description: |
+      Corresponds to the 8th DWORD of the JEDEC Basic Flash Parameter Table,
+      providing erase type 1 and 2 size and instruction.
+  sfdp-basic-9:
+    type: int
+    required: false
+    default: 0x520F
```

Comment:
I'd personally prefer to see these with full 32-bit precision including leading zeros, so it's clear that whatever's in the upper bytes was taken into consideration.  (This should be able to provide the fourth erase size, even though it doesn't by default.)  Strongly encouraged, won't block for it.